src/models/userProfile.js

```diff
@@ -37,6 +37,10 @@ const userProfileSchema = new Schema({
   permissions: {
     frontPermissions: [String],
     backPermissions: [String],
+    removedDefaultPermissions: {
+      front: [String],  // Default permissions removed for this user
+      back: [String]
+    }
```

Comment:
```suggestion
    removedDefaultPermissions: [String]
```
There's only one list of permissions now, so this should be sufficient. `backPermissions: [String]` in the line above is from before and I didn't want to change the DB schema and have to update every entry.

Reply:
I initially thought to make the `removedDefaultPermissions` just a list as well, but didn't because if, in future, differentiating between back and front removed default perms is needed, we don't have to change schema and update entries. If `backPermissions` is never used, it does't make sense to have `removedDefaultPermissions.back`. Should I just remove it?

Reply:
Correct, removing `removedDefaultPermissions.back` is simpler.